api/deployment/get.go

```diff
@@ -85,15 +92,29 @@ func GetDeployment(c *gin.Context) {
 		return
 	}
 
-	// send API call to capture the deployment
-	d, err := scm.FromContext(c).GetDeployment(ctx, u, r, int64(number))
+	// send API call to database to capture the deployment
+	d, err := database.FromContext(c).GetDeployment(int64(number))
 	if err != nil {
-		retErr := fmt.Errorf("unable to get deployment %s: %w", entry, err)
+		// send API call to SCM to capture the deployment
+		dep, err = scm.FromContext(c).GetDeployment(ctx, u, r, int64(number))
+		if err != nil {
+			retErr := fmt.Errorf("unable to get deployment %s: %w", entry, err)
+
+			util.HandleError(c, http.StatusInternalServerError, retErr)
+
+			return
+		}
+	} else {
+		dep = d
+	}
```

Comment:
We can reassign `d` rather than use `dep`: 
```go
	// send API call to database to capture the deployment
	d, err := database.FromContext(c).GetDeployment(int64(number))
	if err != nil {
		// send API call to SCM to capture the deployment
		d, err = scm.FromContext(c).GetDeployment(ctx, u, r, int64(number))
		if err != nil {
			retErr := fmt.Errorf("unable to get deployment %s: %w", entry, err)

			util.HandleError(c, http.StatusInternalServerError, retErr)

			return
		}
	} 
```